### PR TITLE
Fix various issues found by quantifiedcode

### PR DESCRIPTION
**Pull request by nobodyisme**

### CODE REVIEW

File: src/cocaine-app/cache_transport/__init__.py [hunk 1]
```diff
@@ -20,8 +20,11 @@
 
 
 def encode_dict(params):
-    return dict([(k, v if not isinstance(v, unicode) else v.encode('utf-8'))
-                 for k, v in params.iteritems()])
+    return {
+        k: v
+        if not isinstance(v, unicode) else v.encode('utf-8')
```
Comment (shaitan) [inline src/cocaine-app/cache_transport/__init__.py]: This hides that `if/else` is about only `v`, how about using lambda or function here, for example:

``` python
def encode = lambda v:
    v if not isinstance(v, unicode) else v.encode('utf-8')

return {
    k: encode(v)
    for k, v in params.iteritems()
}
```

File: src/cocaine-app/sync/__init__.py [hunk 2]
```diff
@@ -20,8 +20,11 @@
 
 
 def encode_dict(params):
```
Comment (shaitan) [inline src/cocaine-app/sync/__init__.py]: There is such function in `src/cocaine-app/cache_transport/__init__.py`, why do you need both?

**Comment by nobodyisme** [comment]
Removed encode_dict at all, see commit https://github.com/nobodyisme/mastermind/commit/00a545f0438bf90554e2f240974a7eef249314c8

**Comment by shaitan** [comment]
:+1:

**Comment by nobodyisme** [comment]
:+1:

**Pull request merged**